core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -180,7 +180,7 @@ private[chisel3] trait HasId extends InstanceId {
  *
     * @return the current calculation of a name, if it exists
     */
-  private[chisel3] def seedOpt: Option[String] = suggested_seed.orElse(auto_seed)
+  def seedOpt: Option[String] = suggested_seed.orElse(auto_seed)
```

Comment:
Is this intentionally part of this PR, to expose this as a public-facing API?
If so, why?

Reply:
That was a mistake, will fix